druntime/src/core/attribute.d

```diff
@@ -302,3 +302,8 @@ enum mustuse;
  * This is only allowed on `shared` static constructors, not thread-local module constructors.
  */
 enum standalone;
+
+/** 
+ * Use this attribute to indicate that you wish to initialize a field with a mutable reference type.
+ */
+enum singletonInitializer;
```

Comment:
> singletonInitializer

Where does the name come from?

Reply:
The reference you initialize the variable with is a singleton, there is only one instance of it, which is created at compile-time. Do you have a better name? I'm open to ideas

Reply:
`mutableInit`?